ssv/spectest/tests/valcheck/valcheckattestations/valid_non_slashable_slot.go

```diff
@@ -0,0 +1,47 @@
+package valcheckattestations
+
+import (
+	"encoding/hex"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	spec "github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/ssvlabs/ssv-spec/ssv/spectest/tests"
+	"github.com/ssvlabs/ssv-spec/ssv/spectest/tests/valcheck"
+	"github.com/ssvlabs/ssv-spec/types"
+	"github.com/ssvlabs/ssv-spec/types/testingutils"
+)
+
+// ValidNonSlashableSlot tests a valid AttestationData with a slot that is not slashable
+func ValidNonSlashableSlot() tests.SpecTest {
```

Comment:
What is the difference between this and valid?

Reply:
I wanted to create a scenario in which there is a slashable slot and to show that submitting an input for a different slot passes the check.

We had:
- Valid (but no slashable slot)
- Invalid (with exact same slot as the slashable one)

And I created:
- Valid (with a slot that is different from the slashable one).